--- dmenu_orig.c	2012-01-08 06:18:43.000000000 -0600
+++ dmenu.c	2014-04-14 04:19:31.000000000 -0500
@@ -40,7 +40,7 @@
 static void usage(void);
 
 static char text[BUFSIZ] = "";
-static int bh, mw, mh;
+static int bh, mw, mh, ozkar_size=0;
 static int inputw, promptw;
 static size_t cursor = 0;
 static const char *font = NULL;
@@ -100,6 +100,8 @@
 			selbgcolor = argv[++i];
 		else if(!strcmp(argv[i], "-sf"))  /* selected foreground color */
 			selfgcolor = argv[++i];
+        else if (!strcmp(argv[i], "-size"))
+            ozkar_size = atoi(argv[++i]);
 		else
 			usage();
 
@@ -533,7 +535,12 @@
 	utf8 = XInternAtom(dc->dpy, "UTF8_STRING", False);
 
 	/* calculate menu geometry */
-	bh = dc->font.height + 2;
+    if ( ozkar_size != 0) {
+        bh = ozkar_size;
+    } else {
+	    bh = dc->font.height + 2;
+    }
+
 	lines = MAX(lines, 0);
 	mh = (lines + 1) * bh;
 #ifdef XINERAMA
@@ -602,6 +609,6 @@
 void
 usage(void) {
 	fputs("usage: dmenu [-b] [-f] [-i] [-l lines] [-p prompt] [-fn font]\n"
-	      "             [-nb color] [-nf color] [-sb color] [-sf color] [-v]\n", stderr);
+	      "             [-nb color] [-nf color] [-sb color] [-sf color] [-size size] [-v]\n", stderr);
 	exit(EXIT_FAILURE);
 }
